zerver/tests/test_invite.py

```diff
@@ -3000,6 +3001,331 @@ def test_create_multiuse_link_api_call(self) -> None:
         invite_link = self.assert_json_success(result)["invite_link"]
         self.check_user_able_to_register(self.nonreg_email("test"), invite_link)
 
+    def test_edit_multiuse_invite_with_malformed_parameters(self) -> None:
+        """
+        For a successful edit to a multiuse_invite, it is essential
+        that the following parameters are correct.
+        * stream_ids - IDs of the streams the invite applies to
+        * id - ID of the multiuse_invite
+        """
+        realm = get_realm("zulip")
+        iago = self.example_user("iago")
+
+        invite = MultiuseInvite.objects.create(
+            realm=realm,
+            referred_by=iago,
+            include_realm_default_subscriptions=False,
+        )
+
+        initial_invited_as = PreregistrationUser.INVITE_AS["REALM_ADMIN"]
+        modified_invited_as = PreregistrationUser.INVITE_AS["MODERATOR"]
+        initial_streams = [get_stream("Denmark", realm)]
+        modified_streams = [
+            get_stream(stream_name, realm).id for stream_name in ["Denmark", "Verona"]
+        ]
+
+        invite_expires_in_minutes = 2 * 24 * 60
+        do_create_multiuse_invite_link(
```

Comment:
I don't see any reason to call this and `MultiuseInvite.objects.create` both. Just calling this and then `invite = MultiuseInvite.objects.last()` works for me and it should be change for other tests as well.